org.mixedrealitytoolkit.core/Editor/MRTKProjectValidation.cs

```diff
@@ -130,7 +130,7 @@ private static BuildValidationRule GenerateProfileRule(BuildTargetGroup buildTar
                 AssetDatabase.LoadAssetAtPath<MRTKProfile>(AssetDatabase.GUIDToAssetPath(DefaultMRTKProfileGuid))),
                 FixItMessage = $"Assign the default MRTK3 profile for the {buildTargetGroup} build target",
                 Error = false,
-                HelpLink = "https://docs.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-overview/setup#4-configure-mrtk-profile-after-import"
+                HelpLink = "https://learn.microsoft.com/en-us/windows/mixed-reality/mrtk-unity/mrtk3-overview/getting-started/setting-up/setup-new-project#4-configure-mrtk-profile-after-import"
```

Comment:
It might be good to leave the locale code out of the URL to allow it to redirect to the user's setting:

```suggestion
                HelpLink = "https://learn.microsoft.com/windows/mixed-reality/mrtk-unity/mrtk3-overview/getting-started/setting-up/setup-new-project#4-configure-mrtk-profile-after-import"
```

Reply:
That's a great shout, I'll make the change.

Reply:
Changed